omnigibson/robots/r1.py

```diff
@@ -274,7 +302,33 @@ def disabled_collision_pairs(self):
         return [
             ["left_gripper_link1", "left_gripper_link2"],
             ["right_gripper_link1", "right_gripper_link2"],
+            ["base_link", "torso_link1"],
+            ["base_link", "servo_link1"],
+            ["base_link", "wheel_link3"],
+            ["base_link", "servo_link3"],
+            ["base_link", "servo_link2"],
             ["base_link", "wheel_link1"],
             ["base_link", "wheel_link2"],
-            ["base_link", "wheel_link3"],
+            ["servo_link1", "wheel_link1"],
+            ["servo_link2", "wheel_link2"],
+            ["servo_link3", "wheel_link3"],
+            ["torso_link1", "torso_link2"],
+            ["torso_link2", "torso_link3"],
+            ["torso_link3", "torso_link4"],
+            ["torso_link4", "left_arm_link1"],
+            ["left_arm_link1", "left_arm_link2"],
+            ["left_arm_link2", "left_arm_link3"],
+            ["left_arm_link3", "left_arm_link4"],
+            ["left_arm_link4", "left_arm_link5"],
+            ["left_arm_link5", "left_arm_link6"],
+            ["left_arm_link6", "left_gripper_link1"],
+            ["left_arm_link6", "left_gripper_link2"],
+            ["torso_link4", "right_arm_link1"],
+            ["right_arm_link1", "right_arm_link2"],
+            ["right_arm_link2", "right_arm_link3"],
+            ["right_arm_link3", "right_arm_link4"],
+            ["right_arm_link4", "right_arm_link5"],
+            ["right_arm_link5", "right_arm_link6"],
+            ["right_arm_link6", "right_gripper_link1"],
+            ["right_arm_link6", "right_gripper_link2"],
```

Comment:
Do we actually need these? I thought the collision between parent and child links are already disabled by default.